modules/webbrowser/src/webrenderhandler.cpp

```diff
@@ -94,7 +116,72 @@ void WebRenderHandler::OnPaint(CefRefPtr<CefBrowser>, CefRenderHandler::PaintEle
     _needsRepaint = false;
 }
 
+void WebRenderHandler::OnAcceleratedPaint(
+    CefRefPtr<CefBrowser> browser, PaintElementType type, const RectList& dirtyRects,
+    const CefAcceleratedPaintInfo& info
+) {
+    // This should never happen - if accelerated rendering is off the OnPaint method
+    // should be called. But we instatiate the web render handler and the browser instance
+    // in different places so room for error
+    ghoul_assert(_acceleratedRendering, "Accelerated rendering flag is turned off");
+
+    if (dirtyRects.empty()) {
+        return;
+    }
+    // When the window is minimized the texture is 1x1.
+    // This prevents a hard crash when minimizing the window or when you resize the window
+    // so that only the top bar is visible.
+    // TODO ylvse 2024-08-20: minimizing window should be handled with the appropriate
+    // function in the CefBrowser called WasHidden
+    if (dirtyRects[0].height <= 1 || dirtyRects[0].width <= 1) {
+        return; 
+    }
+    // This function is called asynchronously after a reshape which means we have to check
+    // for what we request. Validate the size. This prevents rendering a texture with the
+    // wrong size (the look will be deformed)
+    int newWidth = dirtyRects[0].width;
+    int newHeight = dirtyRects[0].height;
+    if (newWidth != _windowSize.x || newHeight != _windowSize.y) {
+        return;
+    }
+
+    GLuint sharedTexture, memObj;
+    // Create new texture that we can copy the shared texture into. Unfortunately
+    // textures are immutable so we have to create a new one
+    glCreateTextures(GL_TEXTURE_2D, 1, &sharedTexture);
+
+    // Create the memory object handle
+    glCreateMemoryObjectsEXT(1, &memObj);
+
+    // The size of the texture we get from CEF. The CEF format is CEF_COLOR_TYPE_BGRA_8888
+    // It has 4 bytes per pixel. The mem object requires this to be multiplied with 2
+    int size = newWidth * newHeight * 8;
+
+    // Cef uses the GL_HANDLE_TYPE_D3D11_IMAGE_EXT handle for their shared texture
+    // Import the shared texture to the memory object
+    glImportMemoryWin32HandleEXT(
+        memObj, size, GL_HANDLE_TYPE_D3D11_IMAGE_EXT, (void*)info.shared_texture_handle
```

Comment:
```suggestion
        memObj,
        size,
        GL_HANDLE_TYPE_D3D11_IMAGE_EXT,
        reinterpret_cast<void*>(info.shared_texture_handle)
```

Though i'm not sure that the `(void*)` or `reinterpret_cast<void*>` is even necessary since any point should decay into a `void*` automatically